channels_fields/models.py

```diff
@@ -0,0 +1,36 @@
+"""Models for channels_fields"""
+from django.contrib.auth.models import Group
+from django.db import models
+from django.db.models import deletion
+
+from channels.models import BaseChannel
+from channels_fields.constants import FIELD_ROLE_CHOICES
+from course_catalog.models import UserList
+from open_discussions.models import TimestampedModel
+
+
+class FieldChannel(BaseChannel, TimestampedModel):
+    """Field of study"""
+
+    featured_list = models.ForeignKey(
```

Comment:
Do we need this ?

Reply:
A carousel of featured courses is on your MVP field page design.

<img src="https://user-images.githubusercontent.com/125561/172990552-91024d75-e1f8-479b-b094-ebea1f6528c4.png" />

Reply:
<img src="https://user-images.githubusercontent.com/125561/172990564-62e8a6de-6dc3-4e46-8a76-c041e25d7860.png" />

Reply:
I thought we said we'd wait a bit before deciding what to show there and/or how field channel relate to lists

Reply:
I thought that applied to subfields only? It would be much better to have consistent, detailed requirements and designs **before** starting work on implementation. If you want me to take this field out, I'll take it out.